server/lib/src/idm/ldap.rs

```diff
@@ -63,6 +62,12 @@ pub struct LdapServer {
 enum LdapBindTarget {
     Account(Uuid),
     ApiToken,
+    Application(Uuid, Uuid),
```

Comment:
You can do enum structs like:

```
LdapBindTarget {
    ApiToken,
   Application {
       app_id: Uuid,
        user_id: Uuid,
    }
}
```

That way it's a bit better to label/mark fields. The reason we hae enums without this is just historical, that this feature didn't always exist in rust (or I didn't know about it). The woes of project age.